src/services/posts/addComment.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios"
+
+export const addComment = async ({ postId, comment, postsStateDispatch, showToast }) => {
+    try {
+        const res = await axios.post(`/api/posts/comments/${postId}`, { comment });
+        console.log(res);
+        if (res.status === 201) {
+            postsStateDispatch({ type: "UPDATE_POSTS", payload: res.data.posts });
+            showToast({ title: 'comment posted', type: 'success' });
+        }
+    }
+    catch (e) {
+        console.log(e)
+        showToast({ title: e?.response?.errors, type: 'error' });
```

Comment:
Please check if this is working. I can see only the toast, and no message when I try adding the comment.
![image](https://user-images.githubusercontent.com/94738532/167008706-2efc349f-b117-44fd-8a59-4e164f7da84b.png)